tap_helpscout/__init__.py

```diff
@@ -30,12 +28,12 @@ def do_discover():
 def main():
 
     parsed_args = singer.utils.parse_args(REQUIRED_CONFIG_KEYS)
+    if parsed_args.dev:
+        LOGGER.info("Executing tap in dev mode")
```

Comment:
Instead of `info` change it to `warning`. As this is suppose to execute only within dev mode, so if end-user observes this message then he/she should be able to identify some mishap happening.

Reply:
Done